storage/SystemInfo/CmdCryptsetup.cc

```diff
@@ -119,15 +120,123 @@ namespace storage
 	    }
 	}
 
+	switch (encryption_type)
+	{
+	    case EncryptionType::LUKS1:
+		parse_version1(lines);
+		break;
+
+	    case EncryptionType::LUKS2:
+		parse_version2(lines);
+		break;
+
+	    default:
+		y2err("failed to parse luks version in cryptsetup output");
+	}
+
 	y2mil(*this);
     }
 
 
+    void
+    CmdCryptsetupLuksDump::parse_version1(const vector<string>& lines)
+    {
+	static const regex cipher_name_regex("Cipher name:[ \t]*([^ \t]+)[ \t]*", regex::extended);
```

Comment:
And also in all regexps below: the terminating "[ \t]*" part has no effect whatsoever.

Reply:
No, regex_match tries to match the complete string. When using regex_search it would be indeed not needed.

Reply:
Talked to Arvin; the regexps are correct as-is.